module/Motion/CameraAction.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file   CameraAction.cpp
+ * @brief  ミニフィグ撮影動作
+ * @author bizyutyu
+ */
+
+#include "CameraAction.h"
+
+using namespace std;
+
+CameraAction::CameraAction(bool _isClockwis, int _preTargetAngle, int _postTargetAngle)
+  : isClockwis(_isClockwis), preTargetAngle(_preTargetAngle), postTargetAngle(_postTargetAngle){};
```

Comment:
```suggestion
CameraAction::CameraAction(bool _isClockwise, int _preTargetAngle, int _postTargetAngle)
  : isClockwise(_isClockwise), preTargetAngle(_preTargetAngle), postTargetAngle(_postTargetAngle){};
```
他にもたくさん`isClockwis`がいるので修正してください